README.md

```diff
@@ -592,7 +594,7 @@ make
 Optional, this example with Docker.
 
 ```
-docker push <image registry>/<image-name>:<version>
+make push
```

Comment:
Remove this change as #160 is not in, yet.

Reply:
Ok

Reply:
Replaced all `make ... push` lines with `docker push ...`.